OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -918,3 +918,70 @@ TEST_CASE("TableUtilities::resample") {
         CHECK(column[5] == Approx(0.0).margin(1e-10));
     }
 }
+
+TEST_CASE("TimeSeriesTable trimming tests") {
```

Comment:
super nit: including "tests" in the test name is a bit redundant, since it's already implied with `TEST_CASE` in catch.